evalml/preprocessing/utils.py

```diff
@@ -73,11 +73,10 @@ def split_data(X, y, regression=False, test_size=.2, random_state=None):
 
 
 def number_of_features(dtypes):
-    """Get the number of features for specific dtypes.
+    """Get the number of features of each specific dtype in a DataFrame.
```

Comment:
Wanted to clear up this description so users know that it takes in `pd.DataFrame().dtypes`